src/muya/lib/contentState/searchCtrl.js

```diff
@@ -1,26 +1,51 @@
-const defaultSearchOption = {
-  caseSensitive: false,
-  selectHighlight: false,
-  highlightIndex: -1
+import execall from 'execall'
+import { defaultSearchOption } from '../config'
+
+const matchString = (text, value, options) => {
+  const { isCaseSensitive, isWholeWord, isRegexp } = options
+  /* eslint-disable no-useless-escape */
+  const SPECIAL_CHAR_REG = /[\[\]\\^$.\|\?\*\+\(\)\/]{1}/g
+  /* eslint-enable no-useless-escape */
+  let SEARCH_REG = null
```

Comment:
Please use camel case instead upper case or what's the idea behind this?

Reply:
We also use this style to declare RegExp in other files, like `src/muya/lib/config.js`. Just for consistency